app/models/est.rb

```diff
@@ -0,0 +1,35 @@
+class Est
+  include Indexable
+
+  self.settings = {
+    index: {
+      analysis: {
+        analyzer:
+            { custom_analyzer:        {
+              tokenizer: 'standard',
+              filter:    %w(standard asciifolding lowercase snowball) },
```

Comment:
I am not sure if we should use `asciifolding` & `snowball` with portuguese, chinese, and spanish.